articles/azure/azs-how-create-service-principals.md

```diff
@@ -0,0 +1,97 @@
+---
+title: How to create a service principal for Azure AD 
+description: Give applications access to Azure Stack resources by creating service principals
+services: azure-stack
+author: David Woffendin
+toc_rootlink: Users
+toc_sub1: How To
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: How to create a service principal for Azure AD 
+toc_fullpath: Users/How To/azs-how-create-service-principals.md
+toc_mdlink: azs-how-create-service-principals.md
+---
+
+# How to Give applications access to Azure Stack resources
+
+## Overview
+
+In Azure Stack you are able to give an application access to stack resources by creating a service principal that uses Azure Resource Manager. A service principal lets you delegate specific permissions using **role-based access control**. As a best practice, you should use service principals for your applications. Service principals are preferable to running an app using your own credentials for the following reasons:
+
+* You can assign permissions to the service principal that are different than your own account permissions. Typically, a service principal's permissions are restricted to exactly what the app needs to do.
```

Comment:
Put an empty line after each bullet to ensure that the generated html renders correctly

Reply:
done